_data/hosting_providers.json

```diff
@@ -2,12 +2,12 @@
   {
     "name": "Snel.com",
     "link": "https://www.snel.com",
-    "category": "partial",
+    "category": "full",
     "tutorial": "https://www.snel.com/support/set-up-ssl-for-my-website/",
```

Comment:
Do users still need to follow this tutorial for the all-in web hosting plans? If not, we should change this line to the empty string because based on https://github.com/certbot/website#hosting-providers-list, `tutorial` will override the `announcement`/`plan` links.

Reply:
Any thoughts on my comment here?